src/fonduer/supervision/models/label.py

```diff
@@ -1,4 +1,5 @@
 from sqlalchemy import Column, Integer, String
+from sqlalchemy.dialects import postgresql
```

Comment:
Do you think wrap up all postgresSQL specific functions to one place will be easier for user to extend for other databases?

Reply:
This is not easy. We need a major refactor to be able to decouple things cleanly. If we can decouple fonduer and the database, switching to other databases and testing will both be easier. But, that change is definitely out of scope for this PR.